src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -0,0 +1,372 @@
+{
+    "additive": {
+        "attribOptions": ["replace", "sum"]
+    },
+    "accumulate": {
+        "attribOptions": ["none", "sum"]
+    },
+    "alignment-baseline": {
+        "attribOptions": ["after-edge", "alphabetic", "auto", "baseline", "before-edge", "central", "hanging", "ideographic", "mathematical", "middle", "text-after-edge", "text-before-edge"]
+    },
+    "arabic-form": {
+        "attribOptions": ["initial", "isolated", "medial", "terminal"]
+    },
+    "attributeType": {
+        "attribOptions": ["auto", "CSS", "XML"]
+    },
+    "baseline-shift": {
+        "attribOptions": ["baseline", "inherit", "sub", "super"]
+    },
+    "calcMode": {
+        "attribOptions": ["discrete", "linear", "paced", "spline"]
+    },
+    "clip": {
+        "attribOptions": ["auto", "inherit"]
+    },
+    "clipPathUnits": {
+        "alias": "gradientUnits"
+    },
+    "clip-path": {
+        "attribOptions": ["inherit", "none"]
+    },
+    "clip-rule": {
+        "attribOptions": ["evenodd", "inherit", "nonzero"]
+    },
+    "color": {
```

Comment:
@MarcelGerber I have removed all color properties for now to be consistent with CSS hinting. Will add it later including `rebeccapurple` once we get them in CSS too.